### PR TITLE
dotCMS/core#13774 Reload iframes when login logout as happen, fix iframes in content types not loading in FF

### PR DESCRIPTION
**Pull request by fmontes**
dotCMS/core#13774
Incorrect behavior when you are setting permissions and use the login-as functionality
dotCMS/core#13531 Unable to see the content types permissions

### CODE REVIEW

File: src/app/portlets/content-types/layout/content-types-layout.component.spec.ts [hunk 1]
```diff
@@ -55,11 +55,10 @@ class FieldDragDropServiceMock {
     setBagOptions() {}
 }
 
-describe('ContentTypesLayoutComponent', () => {
+fdescribe('ContentTypesLayoutComponent', () => {
```
Comment (alfredo-dotcms) [inline src/app/portlets/content-types/layout/content-types-layout.component.spec.ts]: miss this one

File: src/app/portlets/content-types/layout/content-types-layout.component.spec.ts [hunk 2]
```diff
@@ -150,12 +150,12 @@ describe('ContentTypesLayoutComponent', () => {
         });
     });
 
-    describe('Permission tab', () => {
+    fdescribe('Permission tab', () => {
```
Comment (alfredo-dotcms) [inline src/app/portlets/content-types/layout/content-types-layout.component.spec.ts]: remove the "f" on all the files changed on this commit

File: src/app/view/components/_common/iframe/iframe-component/iframe.component.ts [hunk 3]
```diff
@@ -44,6 +41,12 @@ export class IframeComponent implements OnInit {
         this.iframeOverlayService.overlay.subscribe((val) => (this.showOverlay = val));
         this.element.nativeElement.style.height = this.getIframeHeight(window.innerHeight);
 
+        this.dotIframeService.reloaded().subscribe(() => {
+            if (this.getIframeWindow()) {
```
Comment (freddyucv) [inline src/app/view/components/_common/iframe/iframe-component/iframe.component.ts]: why this?

File: src/app/view/components/_common/iframe/iframe-component/iframe.component.ts [hunk 4]
```diff
@@ -58,20 +61,22 @@ export class IframeComponent implements OnInit {
...
-
-            if (currentPath.indexOf('/c/portal_public/login') !== -1) {
-                this.loginService.logOutUser().subscribe(
-                    (_data) => {},
-                    (error) => {
-                        this.loggerService.error(error);
-                    }
-                );
-            }
+        if (!!this.getIframeWindow() && this.getIframeLocation().pathname.indexOf('/c/portal_public/login') !== -1) {
```
Comment (freddyucv) [inline src/app/view/components/_common/iframe/iframe-component/iframe.component.ts]: maybe we shoud create a know url constant storage somewhere for this ''/c/portal_public/login''

I dont know something like

KNOW_URL = {
    LOGIN: '/c/portal_public/login'
}

File: src/app/portlets/content-types/layout/content-types-layout.component.html [hunk 5]
```diff
@@ -13,14 +13,20 @@ <h4 class="content-type__fields-sidebar-title">{{ i18nMessages['contenttypes.sid
...
+                <dot-iframe [src]="permissionURL"></dot-iframe>
+            </ng-template>
         </p-tabPanel>
-        <p-tabPanel class="content-type__push_history" header="{{ i18nMessages['contenttypes.tab.publisher.push.history.header'] }}" *ngIf="contentTypeId">
-            <dot-iframe [src]="pushHistoryURL"></dot-iframe>
+        <p-tabPanel class="content-type__push_history" header="{{ i18nMessages['contenttypes.tab.publisher.push.history.header'] }}" *ngIf="contentTypeId" [cache]="false">
+            <ng-template pTemplate="content">
+                <dot-iframe [src]="pushHistoryURL"></dot-iframe>
+            </ng-template>
         </p-tabPanel>
```
Comment (freddyucv) [inline src/app/portlets/content-types/layout/content-types-layout.component.html]: can do we test this change anyway?

Comment (fmontes) [reply src/app/portlets/content-types/layout/content-types-layout.component.html]: Is tested... in this [line](https://github.com/dotCMS/core-web/pull/585/files#diff-d8cd5b4e79e98bac770f380cd31252a8R162) you can see that now I have to set the ```selected``` in true in order to get the iframe, if not the iframe is null.

File: src/app/view/components/_common/iframe/iframe-component/iframe.component.ts (hunk 3 above)
Comment (fmontes) [reply src/app/view/components/_common/iframe/iframe-component/iframe.component.ts]: Honestly, not 100% sure, but if I don't do the check first is trow an ```location``` undefined error, not sure if it's something with the iframe, not sure.

File: src/app/view/components/_common/iframe/iframe-component/iframe.component.ts (hunk 4 above)
Comment (fmontes) [reply src/app/view/components/_common/iframe/iframe-component/iframe.component.ts]: It might be cool, but maybe in some of the refactors.

**Pull request merged**